common-app-lib/src/cmr/common_app/api/launchpad_token_validation.clj

```diff
@@ -1,17 +1,66 @@
 (ns cmr.common-app.api.launchpad-token-validation
   "Validate Launchpad token."
   (:require
+   [cheshire.core :as json]
+   [clojure.data.codec.base64 :as base64]
+   [clojure.string :as string]
    [cmr.common-app.config :as config]
    [cmr.common.services.errors :as errors]
    [cmr.transmit.config :as transmit-config]))
 
+;; Note: Similar code exists at gov.nasa.echo.kernel.service.authentication
 (def URS_TOKEN_MAX_LENGTH 100)
 
+(defn is-heritage-token?
```

Comment:
I am hoping we don't invent new generic names for EDL tokens. How about calling this something like `is-prefixed-edl-token?`

Reply:
I combined the heritage and legacy use case, so tokens are not always prefixed. These are tokens that are not JWT or Launchpad